### PR TITLE
Feature/chip styles

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add new stroked/outline variant for chips
- Fix disabled chip styles, adjust current chip styles

#### Screenshots:
Chips After Dark
Chips After Light

#### To Test:
- Check it out here: https://pxb-angular-theme-demo.web.app/material-components/data-display-components

### CODE REVIEW

**Comment by emclaug2** [comment]
FYI there are a couple `!importants` that I have yet to remove.  Ideally our themes should have 0 of these.

**Comment by huayunh** [comment]
image

**Comment by emclaug2** [comment]
Updated styles are pushed out the firebase demo, also removed the !importants.

**Review by daileytj — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## v6.2.0 (Not published yet)
+
+### Added
+
+-   Added new `stroked` variant for `<mat-chip>` and adjusted default chip theme.
+
+### Changed
+
+-   Changed default `<mat-chip>` size to be 18x18 pixels.
```
Comment (daileytj) [inline CHANGELOG.md:11]:
```suggestion
-   Default `<mat-chip>` size to be 18x18 pixels.
```

**Comment by huayunh** [comment]
Were we able to sort out those off-center issues? Avatar font size?

And it would be nice if you may change the margin-left of the icon to 8px (avatar stays 4px).
Under the rule `.pxb-blue .mat-chip .mat-chip-avatar.mat-icon`

image

**Comment by emclaug2** [comment]
> Were we able to sort out those off-center issues? Avatar font size?
> 
> And it would be nice if you may change the margin-left of the icon to 8px (avatar stays 4px).
> Under the rule `.pxb-blue .mat-chip .mat-chip-avatar.mat-icon`
> 
> image

The avatar font size has been adjusted, the margin-left has been adjusted for mat-icon-only mat-avatars.  I've redeployed the demo, could you double check if the off-centered issue is still happening here? 

https://pxb-angular-theme-demo.web.app/material-components/data-display-components

If the content is still off centered let's make a separate issue for this since I've spent some time trying to fix it but just adding a margin offset seems like a hack & flex really should be working here.

**Comment by huayunh** [comment]
Looks all great and lovely now!

**Review by jeffvg — APPROVED** [review]
looks good

**Comment by huayunh** [comment]
> And it would be nice if you may change the margin-left of the icon to 8px (avatar stays 4px).
Under the rule .pxb-blue .mat-chip .mat-chip-avatar.mat-icon

This rule shall work under RTL too

**Comment by emclaug2** [comment]
> > And it would be nice if you may change the margin-left of the icon to 8px (avatar stays 4px).
> > Under the rule .pxb-blue .mat-chip .mat-chip-avatar.mat-icon
> 
> This rule shall work under RTL too

You got it!

**Pull request merged**